### PR TITLE
AOSC OS Core 9.1.1

### LINKED ISSUE #3896: zlib: security update to 1.2.12
### CVE IDs

CVE-2018-25032

### Other security advisory IDs

https://www.debian.org/security/2022/dsa-5111

### Description

commit-msg https://github.com/madler/zlib/commit/5c44459c3b28a9bd3283aaceab7c615f8020c531

### Patches

N/A

### PoC(s)

N/A

### PR DESCRIPTION
**Pull request by CamberLoid**
Topic Description
-----------------

This topic updates AOSC OS Core to 9.1.1, containing a security update of zlib 1.2.12.

Package(s) Affected
-------------------

`zlib` 1.2.11-5 -> 1.2.12
`aosc-aaa` 9.1.0 -> 9.1.1

Security Update?
----------------

Yes #3896 / https://github.com/advisories/GHSA-jc36-42cf-vqwj

Build Order
-----------

```
core-libs/zlib
core-misc/aosc-aaa
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Review by MingcongBai — APPROVED** [review]

**Review by MingcongBai — CHANGES_REQUESTED** [review]
Zlib update results in non-fatal errors:

```
flatpak: /usr/lib/libz.so.1: no version information available (required by /usr/lib/libsoup-2.4.so.1)
flatpak: /usr/lib/libz.so.1: no version information available (required by /usr/lib/libxml2.so.2)
flatpak: /usr/lib/libz.so.1: no version information available (required by /usr/lib/libxml2.so.2)
flatpak: /usr/lib/libz.so.1: no version information available (required by /usr/lib/libgio-2.0.so.0)
```

**Comment by CamberLoid** [comment]
https://github.com/AOSC-Dev/aosc-os-abbs/pull/3903/commits/e681d4e5909579ed6e9c7ef518b13154538a4aab should fix https://github.com/AOSC-Dev/aosc-os-abbs/pull/3903#pullrequestreview-936176582

Discarding label `help-wanted` and preparing for testing.

**Comment by MingcongBai** [comment]
> [e681d4e](https://github.com/AOSC-Dev/aosc-os-abbs/commit/e681d4e5909579ed6e9c7ef518b13154538a4aab) should fix [#3903 (review)](https://github.com/AOSC-Dev/aosc-os-abbs/pull/3903#pullrequestreview-936176582)
> 
> Discarding label `help-wanted` and preparing for testing.

Confirmed.

File: core-libs/zlib/autobuild/build [hunk 1]
```diff
@@ -1,3 +1,6 @@
+abinfo "Building zlib from source ..."
 ./configure --prefix=/usr
```
Comment (MingcongBai) [inline core-libs/zlib/autobuild/build]: Try using ABTYPE, would dropping autobuild/build produce a working build?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Comment by CamberLoid** [comment]
Merging Core 9.1.1 to stable

**Comment by CamberLoid** [comment]
Note: Packages of `riscv64` and `ppcel64` are built in qemu user-mode emulated containers.

**Comment by CamberLoid** [comment]
All packages are pushed to stable.

**Pull request merged**